charmcraft.yaml

```diff
@@ -8,7 +8,7 @@ summary: A Juju charm to operate Kiali
 description: |
   Kiali is a dashboard for Istio, providing visualization and control of your service mesh.
 
-config:
+#config:
```

Comment:
Let's just remove this instead of commenting it out.

Reply:
done.  removed `#provides` too